platform-docs/docs/create-block/attributes.md

```diff
@@ -48,27 +48,27 @@ registerBlockType( 'gutenpride/gutenpride-block', {
 
 ## TextControl Component
```

Comment:
Should this heading be updated, too, since this example is now the `PlainText` component?